tests/System.IO.Compression.Tests/BrotliPerformanceCanterburyCorpus.cs

```diff
@@ -19,19 +19,36 @@ public static IEnumerable<object[]> CanterburyCorpus()
         {
             foreach (CompressionLevel compressionLevel in Enum.GetValues(typeof(CompressionLevel)))
             {
+                string folder = "Canterbury";
```

Comment:
nit: folder doesn't change in this loop. Declare two separate string variables (one for Canterbury folder and one for WebFiles) and move the declarations outside the foreach?